apps/web/src/server/api/routers/product.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from "zod";
+import { createTRPCRouter, publicProcedure } from "~/server/api/trpc";
+
+export const productRouter = createTRPCRouter({
+	getProducts: publicProcedure
+		.input(
+			z.object({
+				limit: z.number().min(1),
+				cursor: z.number().optional(), // Pagination cursor
+			}),
+		)
+		.query(async ({ ctx, input }) => {
+			const { limit, cursor } = input;
+
+			const products = await ctx.db.product.findMany({
```

Comment:
This is great however we would be fetching data from the blockchain instead of the database for the moment and at some point create a routine to bring data from the blockchain to the database and use it as a proxy database.

For the moment let's mock some data and when we have the marketplace contract we can change it to real data.

Reply:
Yes sir. Pleae go ahead and create a mock file for products somwhere inside the router folder and call it in the route for the moment. 

Please leave a TODO comment saying that we'll replace it in the near future.